operations/restrictions.md

```diff
@@ -369,3 +375,113 @@ Removes restrictions from the service.
 ```javascript
 {}
 ```
+
+## Set restrictions
+
+Adds new restrictions with the specified conditions.
+
+If there already exists a restriction with the same conditions, following scenarios apply:
+
+- A. If the exceptions of the new restriction match the old restriction:
+   1) A If the new interval is longer than the old one, new restriction is created joining the two intervals.
```

Comment:
Is it clear what interval is and what's the shortest interval in our system for restrictions? That's how this differs from restrictions/add, no? We added a check for this, might be worth mentioning here that there would be a difference when migration from add.

Reply:
ok, i'll add explanation for the interval here as well